tests/Functional/Action/WebHook/UpdateLineItemsWebhookActionTest.php

```diff
@@ -68,6 +68,37 @@ protected function setUp(): void
         $this->setUpTestLogHandler();
     }
 
+    public function testItReturns401IfNotAuthenticated(): void
```

Comment:
Please change the name of the test case. Because now it's a bit blurry naming for this and the next one. As, requests in both cases would end in `testItReturns401IfNotAuthenticated`.
Proposal: smth like `testItReturns401IfNoCredentialsPassed`.

Reply:
Thanks, fixed